src/BlazorWebView/src/Maui/Windows/SharedSource/WebView2WebViewManager.cs

```diff
@@ -69,9 +69,9 @@ private async Task InitializeWebView2()
 			ApplyDefaultWebViewSettings();
 
 			_webview.CoreWebView2.AddWebResourceRequestedFilter($"{AppOrigin}*", CoreWebView2WebResourceContextWrapper.All);
-			var removeResourceCallback = _webview.CoreWebView2.AddWebResourceRequestedHandler((s, eventArgs) =>
+			var removeResourceCallback = _webview.CoreWebView2.AddWebResourceRequestedHandler(async (s, eventArgs) =>
 			{
-				HandleWebResourceRequest(eventArgs);
+				await HandleWebResourceRequest(eventArgs);
```

Comment:
Should this have a try/catch with any logging?

Reply:
yeah this will just eat any exception no ?